arrow-site/docs/docs/quickstart/README.md

```diff
@@ -266,8 +268,8 @@ following properties:
 ```xml
 
 <properties>
-    <kotlin.version>1.6.10</kotlin.version>
-    <arrow.version>1.0.1</arrow.version>
+    <kotlin.version>1.6.21</kotlin.version>
+    <arrow.version>1.1.1</arrow.version>
```

Comment:
This might be a silly question: shouldn't this be `1.1.0` instead of `1.1.1`?

Reply:
we're updating it to 1.1.1 :D